.github/workflows/ci.yml

```diff
@@ -153,6 +153,28 @@ jobs:
           compile_mode: native
           cflags: "-DMLDSA_DEBUG -fsanitize=address -fsanitize=undefined -fno-sanitize-recover=all"
           ldflags: "-fsanitize=address -fsanitize=undefined -fno-sanitize-recover=all"
+  backend_tests:
+    name: AArch64 FIPS202 backends (${{ matrix.backend }})
+    strategy:
+      fail-fast: false
+      matrix:
+        backend: [x1_scalar, x1_v84a, x2_v84a, x4_v8a_scalar, x4_v8a_v84a_scalar]
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+      - name: build + test
+        uses: ./.github/actions/multi-functest
+        with:
+          nix-shell: 'ci'
+          nix-cache: 'false'
+          gh_token: ${{ secrets.GITHUB_TOKEN }}
+          compile_mode: 'native'
+          opt: 'opt'
+          examples: 'false'
+          cflags: "-DMLDSA_DEBUG -fsanitize=address -fsanitize=undefined -fno-sanitize-recover=all"
+          ldflags: "-fsanitize=address -fsanitize=undefined -fno-sanitize-recover=all"
+          check_namespace: 'false'
```

Comment:
check_namespace is not an argument here. 
CI warns about this: 

```
Unexpected input(s) 'check_namespace', valid inputs are ['nix-shell', 'custom_shell', 'nix-cache', 'nix-verbose', 'gh_token', 'cflags', 'ldflags', 'compile_mode', 'opt', 'func', 'kat', 'acvp', 'examples', 'stack', 'extra_args']
```

Please remove.